RELEASES.md

```diff
@@ -29,7 +29,8 @@ v4.5.0 (development in progress)
     `decreasing_by` for that function come before the `where`. The
     functions in the `where` clause can have their own termination hints, each following the corresponding definition.
   * The `termination_by` clause must bind exactly those parameters that are not
-    already bound by the function header. If there are none, the `=>` must be
+    already bound by the function header, but by a lambda (e.g. `:= fun x y z =>`)
+    or by guards (e.g. `| x, n + 1 => …`). If there are none, the `=>` must be
```

Comment:
Do we call these "guards" in Lean? I thought they were cases. In my mind, a guard is a separate Boolean condition that governs a pattern match case, as e.g. Haskell has.

Reply:
Ah, yes, good point. But what are these things called? The parser calls them `matchAlts`, but there is no `alts` around. In https://github.com/leanprover/lean4/blob/master/doc/declarations.md#the-equation-compiler they are documented, but not actually named. I guess “definition by patterns” at least isn’t wrong.

```suggestion
    or matched by patterns (e.g. `| x, n + 1 => …`). If there are none, the `=>` must be
```

Reply:
`matchAlts` stands for match alternatives, and they are also called match arms. "Definition by cases" also seems like a reasonable description of the syntax.